src/main/java/io/appium/java_client/remote/AppiumCommandExecutor.java

```diff
@@ -192,6 +193,7 @@ private Response createSession(Command command) throws IOException {
 
         ProtocolHandshake.Result result = new AppiumProtocolHandshake().createSession(
                 getClient().with((httpHandler) -> (req) -> {
+                    req.setHeader("User-Agent", AppiumUserAgentFilter.USER_AGENT);
```

Comment:
I think http headers have constants defined in guava lib, which we already use: https://stackoverflow.com/questions/28325180/http-header-field-constant-values

Reply:
Nice, did not notice `com.google.common.net.HttpHeaders`